src/extensions/snippets/SnippetsView/SnippetsRow.tsx

```diff
@@ -0,0 +1,192 @@
+import { GridList, GridListTile } from '@material-ui/core';
```

Comment:
This is pretty much replacing the `SnippetsWidgetBox` class? The code all looks really familiar from the last PR.

Reply:
Yes, mostly a rename